src/eventshub/src/index.css

```diff
@@ -1,3 +1,21 @@
-@tailwind base;
+/* Override Drupal */
+#su-eventshub button {
+  background: unset;
+  color: unset; }
+
+#su-eventshub [type="checkbox"] {
+  clip: unset;
+  border: unset;
+  -webkit-clip-path: unset;
+  clip-path: unset;
+  height: unset;
+  overflow: unset;
+  padding: unset;
+  position: unset;
+  white-space: unset;
+  width: unset;
+}
```

Comment:
This un-does the decanter-core styles but we can look at alternative approaches to doing this.